src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -4761,6 +5071,33 @@ export default class PlotsTab extends React.Component<IPlotsTabProps, {}> {
         );
     }
 
+    @computed get showNoDataTypesSelectedWarning() {
```

Comment:
this function returns a boolean

```suggestion
    @computed get shouldShowNoDataTypesSelectedWarning() {
```